diff -ruN libcrystal-master/src/CMakeLists.txt libcrystal-master-mod/src/CMakeLists.txt
--- libcrystal-master/src/CMakeLists.txt	2018-08-08 20:46:33.000000000 +0800
+++ libcrystal-master-mod/src/CMakeLists.txt	2018-08-22 12:02:58.000000000 +0800
@@ -117,7 +117,7 @@
 endif()

 if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
-    set(SYSTEM_LIBS pthread m)
+    set(SYSTEM_LIBS pthread m rt)
 elseif(WIN32)
     set(SYSTEM_LIBS Ws2_32 Winmm pthread)
 endif()
